rpi/board.go

```diff
@@ -192,13 +201,46 @@ func (pi *piPigpio) Reconfigure(
 		return err
 	}
 
+	pi.reconfigurePulls(ctx, cfg)
+
 	boardInstanceMu.Lock()
 	defer boardInstanceMu.Unlock()
 	boardInstance = pi
 
 	return nil
 }
 
+func (pi *piPigpio) reconfigurePulls(ctx context.Context, cfg *Config) {
```

Comment:
should we not swallow the error? It would be good to surface this from within the reconfigure.

Reply:
sure i can have reconfigure fail instead